[datadog_integration_azure] Fix docs for app_service_plan_filters (#2085)
DataDog/terraform-provider-datadog: docs/resources/integration_azure.md

### Optional

> - `app_service_plan_filters` (String) This comma separated list of tags (in the form key:value) defines a filter that we will use when collecting metrics from Azure App Service Plans. Only App Service Plans that match one of the defined tags will be imported into Datadog. The rest, including the apps and functions running on them, will be ignored. Note that this will also filter the metrics for any App or Function running on the App Service Plan(s).

Reviewer: ```suggestion
- `app_service_plan_filters` (String) This comma-separated list of tags (in the form `key:value`) defines a filter that Datadog uses when collecting metrics from Azure App Service Plans. Only App Service Plans that match one of the defined tags is imported into Datadog. The rest, including the apps and functions running on them, are ignored. This also filters the metrics for any App or Function running on the App Service Plan(s).
```
